docs/getting_started.rst

```diff
@@ -0,0 +1,296 @@
+    dsm.call_genotype
+
+    # Count alternate alleles while omitting partial calls
+    ##############
+    # Using Xarray
+    ##############
+    import xarray as xr
+    alt_allele_count = xr.where(
+        # Identify where there are any missing calls across chromosomes
+        dsm.call_genotype_mask.any(dim='ploidy'),
+        -1, # Return -1 if any one call for a chromosome is missing
+        (dsm.call_genotype > 0).sum(dim='ploidy') # Otherwise, sum non-ref calls
+    )
+    # Note that only the first two samples have meaningful counts since
+    # at least one call is missing for the last two samples
+    alt_allele_count.values
+
+    #############
+    # Using Numba
+    #############
+    import numba
+    import numpy as np
+
+    def alt_allele_count(gt):
+        out = np.full(gt.shape[:2], -1, dtype=np.int64)
+        for i, j in np.ndindex(*out.shape):
+            if np.all(gt[i, j] >= 0):
+                out[i, j] = np.sum(gt[i, j] > 0)
+        return out
+
+    # Jit-compiled functions are often simpler with a single array input, since
+    # conditional logic based on sentinel values is easier to program with this API
+    numba.njit(alt_allele_count)(dsm.call_genotype.values)
+
+This is not necessarily a level of detail most users should need to worry about. Missing data
+is handled explicitly in sgkit functions and where this isn't possible, limitations related
+to it are documented along with potential workarounds.
+
+Chaining operations
+~~~~~~~~~~~~~~~~~~~
+
+This example shows to chain multiple sgkit, xarray, and pandas operations into a single pipeline:
+
+.. ipython:: python
+
+    # Use `pipe` to apply a single sgkit function to a dataset
+    ds_qc = ds.pipe(sg.variant_stats).drop_dims('samples')
+    ds_qc
+
+    # Show statistics for one of the arrays to be used as a filter
+    ds_qc.variant_call_rate.to_series().describe()
+
+    # Build a pipeline that filters on call rate and computes Fst between two populations
+    (
+        ds
+        # Add call rate and other statistics
+        .pipe(sg.variant_stats)
+        # Apply filter to include variants present across > 80% of samples
+        .pipe(lambda ds: ds.sel(variants=ds.variant_call_rate > .8))
+        # Assign a "cohort" variable that splits samples into two groups
+        .assign(sample_cohort=np.repeat([0, 1], ds.dims['samples'] // 2))
+        # Compute Fst between the groups
+        # TODO: Refactor based on https://github.com/pystatgen/sgkit/pull/260
+        .pipe(lambda ds: sg.Fst(*(g[1] for g in ds.groupby('sample_cohort'))))
+        # Extract the single Fst value from the resulting array
+        .item(0)
+    )
+
+This is possible because sgkit functions nearly always take a ``Dataset`` as the first argument, create new
+variables, and then merge these new variables into a copy of the provided dataset in the returned value.
+See :ref:`dataset_merge` for more details.
+
+Chunked arrays
+~~~~~~~~~~~~~~
+
```

Comment:
A quick intro on what chunked arrays are with links to upstream docs would be super helpful here.

Reply:
https://github.com/pystatgen/sgkit/pull/278/commits/ae349eea8e8b2094f7d55bf282e5c15d74c4a84b#diff-9bc8bf6e8d9db020cefce1fbc0426795R305